contrib/Dockerfile.manylinux

```
@@ -226,10 +226,12 @@ RUN git clone https://github.com/nvidia/gusli.git && \
      cd ..
 
 # Required for Azure SDK
-RUN git clone --depth 1 https://gitlab.gnome.org/GNOME/libxml2.git --branch 2.15 && \
-    cd libxml2 && \
+RUN wget -q --tries=3 --waitretry=5 https://download.gnome.org/sources/libxml2/2.15/libxml2-2.15.3.tar.xz && \
```

Hadolint (2.14.0)

[warning] Use WORKDIR to switch to a directory

(DL3003)